src/main/java/io/appium/java_client/remote/MobileOptions.java

```diff
@@ -0,0 +1,515 @@
+     */
+    public T setLocale(String locale) {
+        return amend(MobileCapabilityType.LOCALE, locale);
+    }
+
+    /**
+     * Get locale abbreviation for use in session.
+     *
+     * @return String representing the locale abbreviation.
+     * @see MobileCapabilityType#LOCALE
+     */
+    public String getLocale() {
+        return (String) getCapability(MobileCapabilityType.LOCALE);
+    }
+
+    /**
+     * Set the timeout for new commands.
+     *
+     * @param seconds is the number of seconds to timeout before seeing a new command.
+     * @return this MobileOptions, for chaining.
+     * @see MobileCapabilityType#NEW_COMMAND_TIMEOUT
+     */
+    public T setNewCommandTimeout(Duration seconds) {
+        return amend(MobileCapabilityType.NEW_COMMAND_TIMEOUT, seconds);
+    }
+
+    /**
+     * Get the timeout for new commands.
+     *
+     * @return timeout before seeing a new command in seconds.
+     * @see MobileCapabilityType#NEW_COMMAND_TIMEOUT
+     */
+    public Duration getNewCommandTimeout() {
+        return (Duration) getCapability(MobileCapabilityType.NEW_COMMAND_TIMEOUT);
+    }
+
+    /**
+     * Set the app not to do a reset.
+     *
+     * @return this MobileOptions, for chaining.
+     * @see MobileCapabilityType#NO_RESET
+     */
+    public T setNoReset() {
+        return setNoReset(true);
+    }
+
+    /**
+     * Set whether the app does not do a reset.
+     *
+     * @param bool is whether the app does not do a reset.
+     * @return this MobileOptions, for chaining.
+     * @see MobileCapabilityType#NO_RESET
+     */
+    public T setNoReset(boolean bool) {
+        return amend(MobileCapabilityType.NO_RESET, bool);
+    }
+
+    /**
+     * Get whether the app does not do a reset.
+     *
+     * @return true if the app does not do a reset.
+     * @see MobileCapabilityType#NO_RESET
+     */
+    public boolean doesNoReset() {
+        return (boolean) getCapability(MobileCapabilityType.NO_RESET);
+    }
+
+    /**
+     * Set the orientation of the screen.
+     *
+     * @param orientation is the screen orientation.
+     * @return this MobileOptions, for chaining.
+     * @see MobileCapabilityType#ORIENTATION
+     */
+    public T setOrientation(ScreenOrientation orientation) {
+        return amend(MobileCapabilityType.ORIENTATION, orientation);
+    }
+
+    /**
+     * Get the orientation of the screen.
+     *
+     * @return ScreenOrientation of the app.
+     * @see MobileCapabilityType#ORIENTATION
+     */
+    public ScreenOrientation getOrientation() {
+        return (ScreenOrientation) getCapability(MobileCapabilityType.ORIENTATION);
+    }
+
+    /**
+     * Set the location of the app(s) to install before running a test.
+     *
+     * @param apps is the apps to install.
+     * @return this MobileOptions, for chaining.
+     * @see MobileCapabilityType#OTHER_APPS
+     */
+    public T setOtherApps(List<String> apps) {
+        return amend(MobileCapabilityType.OTHER_APPS, apps);
```

Comment:
this should be a comma separated string

Reply:
1df2efd08b737ca1c942d244dc17ae2cd92b9d97